### DISCUSSION TITLE
File storage error

### DISCUSSION THREAD

**Discussion by thundersparkf** [chatwoot/chatwoot#2558] [Q&A] [answered]
While sending files over Twilio WhatsApp(in development), this error keeps popping up which causes the files to not be recieved at all. 
<img width="1041" alt="Screenshot 2021-07-02 at 5 29 43 PM" src="https://user-images.githubusercontent.com/64605024/124279010-981a3900-db64-11eb-8eee-f423af543624.png">

Is this a some local storage settings issue? Have hit a roadblock here.

Update: Cool so apparently I needed to run
```
rails active_storage:update
rails db:migrate
```
 Need to know if this is the solution cause this changes the schema.

<img width="905" alt="Screenshot 2021-07-03 at 10 44 47 AM" src="https://user-images.githubusercontent.com/64605024/124343769-baa36500-dbeb-11eb-89e9-9b4dafb2ada0.png">

<img width="971" alt="Screenshot 2021-07-03 at 10 47 14 AM" src="https://user-images.githubusercontent.com/64605024/124343805-1241d080-dbec-11eb-81e8-87fc22074f74.png">

**Comment by sojan-official** [comment, marked as answer]
@thundersparkf  you might be working with a newer version of rails than what chatwoot uses. The newer versions of rails have this migration to add the attribute. `service_name` to active storage tables. 

ideally, start off with the `develop` branch of chatwoot. Do a `bundle install` so that the rails version specified in the gem file will be used.

**Reply by thundersparkf** [reply]
Ah dip! Thanks tho however I was little too eager and submitted a PR with the updated version. Sorry about that!
